test/kreflection/test-kreflection.sh

```diff
@@ -0,0 +1,9 @@
+TOP=$(git rev-parse --show-toplevel)
```

Comment:
You must not set `TOP`, `KRUN`, `KORE_EXEC`, or `KRUN_OPTS` here. These must be set by `make`. If a variable you need is not set by `make`, add an `export` declaration to the `Makefile`, such as: `export KRUN`.